pkgs/desktops/deepin/go-package/dde-daemon/default.nix

```diff
@@ -0,0 +1,229 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, replaceAll
+, getUsrPatchFrom
+, buildGoPackage
+, pkg-config
+, deepin-gettext-tools
+, gettext
+, python3
+, wrapGAppsHook
+, go-dbus-factory
+, go-gir-generator
+, go-lib
+, dde-api
+, alsa-lib
+, glib
+, gtk3
+, libgudev
+, libinput
+, libnl
+, librsvg
+, linux-pam
+, networkmanager
+, pulseaudio
+, glibc
+, gdk-pixbuf-xlib
+, tzdata
+, xkeyboard_config
+, runtimeShell
+, fprintd
+, xorg
+, xdotool
+, dbus
+, getconf
+, util-linux
+, ddcutil
+, libxcrypt
+}:
+let
+  patchList = {
+    "gesture/config.go" = [
+      # "/usr/share/dde-daemon/gesture.json"
+    ];
+    "appearance/fsnotify.go" = [
+      [ "/usr/share" "/run/current-system/sw/share" ]
```

Comment:
We really shouldn't hardcode `/run/current-system` here, can we patch it to use something like `XDG_DATA_DIRS`